src/app/component/core/nav-bar/nav-bar.component.ts

```diff
@@ -83,6 +84,10 @@ export class NavBarComponent implements OnInit {
     });
   }
 
+  public openModalSignIn(){
```

Comment:
Please, format your code and specify the return type

Reply:
ready